jmix-flowui/flowui/src/main/java/io/jmix/flowui/facet/urlqueryparameters/GenericFilterUrlQueryParametersBinder.java

```diff
@@ -76,6 +77,7 @@ public class GenericFilterUrlQueryParametersBinder extends AbstractUrlQueryParam
     protected String conditionParam;
 
     protected ApplicationContext applicationContext;
+    private final MetadataTools metadataTools;
```

Comment:
1. we don't use private
2. follow code style of the class

Reply:
Fixed. The field was removed.